README.md

```diff
@@ -190,30 +190,47 @@ Apps people use when taking transit.
 
 Software that makes it easy to consume GTFS data in a variety of languages.
 
-- [Mapzen GTFS](https://github.com/transitland/mapzen-gtfs) - A Python GTFS library that supports reading individual GTFS tables, or constructing a graph to represent each agency in a feed.
-- [gtfsdb](https://github.com/OpenTransitTools/gtfsdb) - Python library for converting GTFS files into a relational database.
+**C**
```

Comment:
@mesozoic-drones Thanks for working on this! Conceptually I agree that this would make the list easier to search. One request - could you stick to the markdown formatting for headers instead of bolding the language names?

So:

```
#### C
```

...instead of:

```
**C**
```

Reply:
Done.